### DISCUSSION TITLE
SEARCH FOR MULTIPLES SKUS

### DISCUSSION THREAD

**Discussion by lucasbayer22** [Smile-SA/elasticsuite#3153] [Q&A]
Hey guys,

All my products have skus like 307985, 177087, 368357.

How do I need to configure the Elastic Suite module to return these three products when my search criteria is 307985, 177087, 368357.

**Comment by rbayet** [comment]
Hello @lucassilva15,

Could you clarify a bit the context ?
That seems a bit weird that someone would search for three distinct SKUs in the frontend, is this a regular situation in your shop ?
Is there any "link" between those products, are they just variation of a same product, like, say, electric engines of different power output ?

Regards,
